src/storage/swiftOS.py

```diff
@@ -251,6 +251,42 @@ def create_upload_form(self,sourcepath,containername,prefix,ttl,max_file_size):
 
         return retval
 
+    def list_objects(self,containername,prefix=None):
+        # object_prefix = "{prefix}/{objectname}".format(prefix=prefix, objectname=objectname)
+
+        url = f"{self.url}/{containername}"
+
+        print(url)
```

Comment:
Should this `print` statement be deleted?

Reply:
removed